src/Optimizer/Transformer/GoogleFontsPreconnect.php

```diff
@@ -0,0 +1,67 @@
+<?php
```

Comment:
Rather than providing a transformer for one custom font, would we want to create a generic `FontsPreconnect` transformer that supports adding resource hints for any of the custom fonts on the allowlist?

(See related [validation rule](https://github.com/ampproject/amphtml/blob/f2a49b8efa4ecb0cd1dd273977799299a9e1f816/validator/validator-main.protoascii#L349-L391))

Reply:
This may not be necessary because I think the unique issue with Google fonts is that the font files are located on `https://fonts.gstatic.com` whereas the stylesheets that reference the fonts are on `https://fonts.googleapis.com`, so adding the preconnect to `https://fonts.gstatic.com` will ensure that the font files are loaded fast once the stylesheet is loaded. 

In contrast, looking at Font Awesome, I see the font stylesheet example of https://use.fontawesome.com/releases/v5.8.1/css/all.css references font files which are on the same domain. Therefore no preconnect is needed in that case.